frundenbot/notifier.py

```diff
@@ -0,0 +1,52 @@
+from emoji import emojize
+from telegram import ParseMode
+
+from frundenbot import STATE_OPEN, STATE_UNKNOWN, MESSAGE_OPEN
+
+
+class Notifier:
+    """
+    Used to send notifications when the "open" status changes
+    """
+
+    # set of chat_ids that have registered for a notification
+    chat_ids = set()
+    # the last
```

Comment:
This looks a bit like you wanted to write more? :thinking:

Reply:
obsolete